org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/repositories/wizards/AbapGitWizardBranchSelection.java

```diff
@@ -0,0 +1,182 @@
+package org.abapgit.adt.ui.internal.repositories.wizards;
```

Comment:
Move the classes to the package org.abapgit.adt.ui.internal.wizards.
Technically it's right to have repositories in the package name, but presently all wizards & related pages are only related to repositories view.

In future in case we have wizards also for staging view, in that case we can have this new package.